activestorage/test/models/preview_test.rb

```diff
@@ -4,6 +4,8 @@
 require "database/setup"
 
 class ActiveStorage::PreviewTest < ActiveSupport::TestCase
+  PREVIEW_METHODS = %i[key url download]
```

Comment:
@jonathanhefner Since these methods are public method and public interface as well, I wanted to test them each individually even though they are similar.

I believe in the future, adding more public methods to AS:Preview would simply require adding the method under test to this constant.

Right now I believe I'm testing 2/3 of these methods. the 1/3 left is when provided a non blank transformation. I don't know if you think it's worth testing.

Reply:
I decided to add the missing third. And with this, I believe we have full coverage of all of the three preview methods.